slade.rb

```diff
@@ -26,7 +26,7 @@ class Slade < Formula
   depends_on "wxwidgets-slade"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    system "cmake", "-DGLEW_INCLUDE_PATH=/usr/local/Cellar/glew/2.1.0/include", "-DGLEW_LIBRARY=/usr/local/Cellar/glew/2.1.0/lib/libGLEW.2.1.dylib", ".", *std_cmake_args
```

Comment:
> INCLUDE_PATH and _LIBRARY might not work, guessed their paths.

Could you please tell me why those paths were not detected by CMake?
I think, that problem needs to be fixed instead of hardcoding a version.
Also, GLEW is still being developed, just check its commit history.

Reply:
Yeah, I agree, 

Not sure why CMake didn't find them, but I was getting errors about it not finding GLEW before I put in those arguments.

I'll look into this when I have some time.